### ISSUE TITLE
JS minification doesn't work

### ISSUE THREAD

**Issue by Samuel91** [abagshaw/brackets-minifier#27]
Since last update for me js minification doesn't work at all.

**Comment by abagshaw** [comment]
1. What OS are you on?
2. If you have made any changes to the minifier preferences, can you copy and page the `.brackets.json` file in your project's root?
3. Can you copy and paste any errors that appear in the Developer Tools Console (_Help_ -> _Show Developer Tools_ -> _Console_)?

**Comment by greenpharey** [comment]
I'm having the same issue with the js minification not working.

- OS: Windows 10

- My brackets.json file:
`{
    "brackets-eslint.gutterMarks": true,
    "brackets-eslint.useLocalESLint": false,
    "fonts.fontSize": "14px",
    "fonts.fontFamily": "'SourceCodePro-Medium', ＭＳ ゴシック, 'MS Gothic', monospace",
    "extensions.sort": "downloadCount",
    "brackets-minifier.on-save": true,
    "themes.theme": "monokai-strikes-back",
    "custom.work.useIconsOfCustomWork": true,
    "custom.work.blueish": true,
    "custom.work.moveToolbarToSidebar": true,
    "custom.work.autoHideTheTabmenu": true,
    "linting.collapsed": true
}`

- I don't see any errors in the Developer Tooks Console.

**Comment by abagshaw** [comment]
@greenpharey Can you paste the contents the `.brackets.json` file that is in the root of your project (if you made any changes to the minifier preferences). What you pasted above is from the main `.brackets.json` for global settings across the whole Brackets application.

When you say js minification is "not working" can you be more specific? Does any message show up in the lower right, is the minified file created at all?

**Comment by greenpharey** [comment]
@abagshaw - Sorry about that.  I didn't make any changes to the minifier preferences.

For me, when I right-click and choose 'Minify" or click Ctrl+Alt+M, the only thing that happens in the lower right corner is an extra box appears that says "Minified" that's written in black text so I can barely see it.  Other than that nothing else happens on the screen. I also never see the js min file appear in my file respository.

**Comment by abagshaw** [comment]
@greenpharey Could you do a search on your computer for `filename.min.js` where `filename` is the original filename of the unminified JS file? There's a possibility that the path isn't being set properly and the minified file is being saved somewhere weird.

Also, this is a separate issue, but are you using some kind of dark theme for brackets - brackets-minifier wasn't designed to adapt the status message to different backgrounds (it assumed the text was placed on the light background) - but if there's a way to detect a dark background then I can see if that can be fixed.

**Comment by greenpharey** [comment]
Hi @abagshaw  - I did the search via your instructions and it says the file was in the correct path even though it was not appearing on the screen.  I refreshed the file tree a couple of times and then it appeared.  I minified another file and it did not appear at first until i refreshed the file tree a couple of times. Maybe there is some sort of lag. 

With respect to the dark status message, yes I do use a dark background.  ""themes.theme": "monokai-strikes-back"," from my global backets.json is the theme I use. It looks like when there's a ".dark class on the body tag then the following code is added to the brackets.min.css file:
`html .dark, body .dark, html.dark, body.dark {
    color: #ccc;
}`

Perhaps, when that .dark class exists on the body tag, you could apply the same color to your status message? Or remove the color that's overwriting it in general so that it accepts what is default?

**Comment by abagshaw** [comment]
@greenpharey I've actually noticed the same problem on my machine - with brackets not updating the file tree. If you look in the folder (with File Explorer) you can see that the minified file shows up immediately upon minification, brackets just doesn't seem to want to display it sometimes. I'm not really sure if this is something wrong with the plugin - my guess is that it's an issue with brackets.

In regards to the dark mode issue - looks like that should be a fairly straightforward fix. I'll try to get around to that when I have some time.

**Comment by Samuel91** [comment]
1) OS high sierra
2) no changes in the minifier preferences
3) no errors are shown in the Developer Tool Console

**Comment by abagshaw** [comment]
@Samuel91 Does anything show up in the lower right hand corner of the Brackets window when you try to minify the file? Also, can you navigate to the location of where the minified file should be to see if this is a problem similar to the one @greenpharey was talking about?

**Comment by Samuel91** [comment]
@abagshaw I don't know why but today without any upgrades it is working correctly again but the shortcut for minification doesn't work!??

**Comment by abagshaw** [comment]
@Samuel91 That's odd, are you saing that if you manually trigger minification with _Edit_ -> _Minify_ does it works, but using the shortcut `Cmd+Alt+M` doesn't do anything?

Can you check the dev console again for any errors?

**Comment by Samuel91** [comment]
Exactly! I checked again but no errors are shown in the  dev console.

**Comment by abagshaw** [comment]
@Samuel91 Have you installed/updated any other Brackets extensions recently that may also be trying to use the `Cmd+Alt+M` shortcut?

**Comment by Samuel91** [comment]
I don't think so, i simple updated the ones I had before. How can I check this?

**Comment by SethClydesdale** [comment]
Hi,

I have the same issue with this plugin. Whenever I try to minify my JS via CTRL+ALT+M or Context>Minify I get the following error.
> "timeout" @ main.js:237

I got this error earlier, so I reinstalled the plugin and it worked. However, after starting brackets up again I'm facing the same issue.

I'm on Win10 btw, if that helps.
